Source/DafnyCore/Verifier/Statements/BoogieGenerator.TrPredicateStatement.cs

```diff
@@ -77,14 +77,15 @@ public void TrAssertStmt(PredicateStmt stmt, BoogieStmtListBuilder builder, List
       BoogieStmtListBuilder proofBuilder = null;
       var assertStmt = stmt as AssertStmt;
       if (assertStmt != null) {
-        if (assertStmt.Proof != null) {
-          hiddenProof = true;
-          proofBuilder = new BoogieStmtListBuilder(this, options, builder.Context);
-          AddComment(proofBuilder, stmt, "assert statement proof");
-          CurrentIdGenerator.Push();
-          TrStmt(((AssertStmt)stmt).Proof, proofBuilder, locals, etran);
-          CurrentIdGenerator.Pop();
-        } else if (assertStmt.Label != null) {
+        // if (assertStmt.Proof != null) {
+        //   hiddenProof = true;
+        //   proofBuilder = new BoogieStmtListBuilder(this, options, builder.Context);
+        //   AddComment(proofBuilder, stmt, "assert statement proof");
+        //   CurrentIdGenerator.Push();
+        //   TrStmt(((AssertStmt)stmt).Proof, proofBuilder, locals, etran);
+        //   CurrentIdGenerator.Pop();
+        // } else 
```

Comment:
Should this also be removed, rather than commented out?

Reply:
already done